python/ql/src/Security/CWE-295/examples/paramiko_host_key.py

```diff
@@ -0,0 +1,9 @@
+from paramiko.client import SSHClient, AutoAddPolicy
+
+client = SSHClient()
+client.set_missing_host_key_policy(AutoAddPolicy)
+client.connect("example.com")
+
+# ... interaction with server
+
+client.close()
```

Comment:
Should have a "good" example as well, and encapsulate them in functions, so it is clear which is which.